gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/java/io/gravitee/repository/jdbc/management/JdbcEventRepository.java

```diff
@@ -47,30 +47,41 @@
 import java.util.Map.Entry;
 import java.util.Optional;
 import java.util.Set;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.jdbc.core.BatchPreparedStatementSetter;
-import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.stereotype.Repository;
 import org.springframework.util.CollectionUtils;
 
 /**
  *
  * @author njt
  */
+@Slf4j
 @Repository
 public class JdbcEventRepository extends JdbcAbstractPageableRepository<Event> implements EventRepository {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(JdbcEventRepository.class);
-    private final String EVENT_PROPERTIES;
-    private final String EVENT_ENVIRONMENTS;
+    protected final String EVENT_PROPERTIES;
```

Comment:
```suggestion
    private final String EVENT_PROPERTIES;
```

Reply:
Was an aborted attempt to make JdbcEventLatestRepository inherit JdbcEventRepository 😅